sktime/transformations/series/boxcox.py

```diff
@@ -123,12 +123,17 @@ class BoxCoxTransformer(BaseTransformer):
         is applied to the absolute value while the sign is kept.
         If ``False``, any negative values will be passed unchanged to the
         underlying functions (possibly causing error).
+    adjust_bias : bool, optional, default = False
+        If True, use the back-transformed mean method instead of simple back-transform.
```

Comment:
The location in fpp should be cited here, and what True exactly does in transform and inverse_transform.